translations/base-es.yaml

```diff
@@ -43,24 +43,24 @@ steamPage:
 
         [list]
             [*] Puntos de referencia en el mapa
-            [*] Ilimitadas partidas guardadas
+            [*] Partidas guardadas ilimitadas
             [*] Modo nocturno
             [*] Más opciones
             [*] Permitirme seguir desarrollando shapez.io ❤️
             [*] Más características en el futuro!
         [/list]
 
-        [b]Características planeadas & sugerencias de la comunidad[/b]
+        [b]Características planeadas y sugerencias de la comunidad[/b]
 
         Este juego es de código abierto - Cualquiera puede contribuir! A parte de eso, escucho [b]mucho[/b] a la comunidad! Intento leer todas las sugerencias e intento tener en cuenta todo el foodback posible.
         [list]
-            [*] Modo historia en el que los edificios cuesten figuras
+            [*] Modo historia en el que los edificios tengan un coste de figuras
             [*] Más niveles y edificios (exclusivos del juego completo)
             [*] Mapas diferentes y tal vez obstáculos en el mapa
             [*] Configuración en la creación del mapa (Editar el número y tamaño de los recursos, la semilla, y más)
             [*] Más tipos de formas
             [*] Mejoras de rendimiento (Aunque el juego ya funciona muy bien!)
-            [*] Modo para daltónicos
+            [*] Modo para el daltonismo
```

Comment:
"Modo para daltónicos"  is the best traduction, "para el daltonismo" feels weird, is not a mode for the condition, but for the people who has the condition.